docs/developer-guide/extensions.md

```diff
@@ -117,3 +117,54 @@ You will find both the index.json file and a sample translation folder in build/
     ]
 }
 ```
+
+### Extensions uploading service
+
+Extensions can be uploaded using the context creator UI of MapStore. The storage and configuration of the uploaded zip bundle as
+a proper extension is managed by a dedicated MapStore backend service, the ***Upload Service***.
+The Upload Service is responsible of unzipping the bundle, storing javascript and the other extension assets in the extensions folder and updating the configuration files needed by MapStore to use the extension:
+
+ * extensions.json (the extensions registry)
+ * pluginsConfig.json (the context creator plugins catalog)
+
+### Extensions and datadir
+
+Extensions work better if you use a [datadir](externalized-configuration), because when a datadir is configured,
```

Comment:
This link is broken

Reply:
@ElenaGallo are you able to fix it quickly?

Reply:
[Here](https://github.com/geosolutions-it/MapStore2/pull/5274) the fix